nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/lossfunctions/SDLossMAE.java

```diff
@@ -0,0 +1,17 @@
+package org.nd4j.linalg.lossfunctions;
+
+import org.nd4j.autodiff.samediff.SDVariable;
+import org.nd4j.autodiff.samediff.SameDiff;
+
+public class SDLossMAE extends SameDiffLoss {
+    public SDLossMAE(){
+        super();
+    }
```

Comment:
(c) the superclass constructor call `super();` is implied (i.e., Java will call that anyway, whether you include it or not - so don't include it).
In fact the whole no-arg constructor (SDLossMAE() and SDLossMSE()) is implied in that way too - if no other contsructor is defined, a public no-arg constructor is created by default. We can delete these constructors and let java automatically generate them, in other words.